ros/src/waypoint_updater/CMakeLists.txt

```diff
@@ -162,10 +164,12 @@ include_directories(
 
 ## Mark executable scripts (Python etc.) for installation
 ## in contrast to setup.py, you can choose the destination
-# install(PROGRAMS
-#   scripts/my_python_script
-#   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-# )
+#catkin_install_python(PROGRAMS nodes/updater.py
+#  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+#install(PROGRAMS
+#  scripts/waypoint_updater.py
+#  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+#)
```

Comment:
There are a lot of changes to lines that are just commented out.  Do you know why these changed?  Do they need to change?